src/main/java/pl/niewiemmichal/underhiseye/model/Visit.java

```diff
@@ -0,0 +1,54 @@
+package pl.niewiemmichal.underhiseye.model;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@RequiredArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Visit {
+
+    @Id @GeneratedValue
+    private Integer id;
+
+    @Lob
+    private String description;
+
+    @Lob
+    private String diagnosis;
+
+    @NonNull
+    @Column(nullable = false)
+    private Integer status;
+
+    @NonNull
+    @Column(nullable = false)
+    @Temporal(TemporalType.DATE)
+    private Date registrationDate;
+
+    @NonNull
+    @Column(nullable = false)
+    @Temporal(TemporalType.DATE)
+    private Date date;
+
+    @NonNull
+    @JoinColumn(nullable = false)
+    @ManyToOne
+    private Patient patient;
+
+    @NonNull
+    @JoinColumn(nullable = false)
+    @ManyToOne
+    private PatientRegistrationSpecialist registrationSpecialist;
+
+    @NonNull
+    @JoinColumn(nullable = false)
+    @ManyToOne
+    private Doctor doctor;
```

Comment:
Ask @ClaudiaPolaczek if this should be nullable or not.

Reply:
As a patient you are being registered to see a specific doctor. For example you may want to see an oncologist or maybe a dermatologist so the system has to keep an eye on every planned visit in order to  provide you with information about nearest date and hour you will be able to consult your health concerns,